ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -323,9 +357,166 @@ func (ipfs *Connector) pinLsHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	respBytes, _ := json.Marshal(pinLs)
+	resBytes, _ := json.Marshal(pinLs)
 	w.WriteHeader(http.StatusOK)
-	w.Write(respBytes)
+	w.Write(resBytes)
+}
+
+func (ipfs *Connector) addHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Looked at it, if there is no better way, let's do it like that. I would be for adding additional message to go-ipfs (containing what is getting pinned) it could be used for verbose mode of ipfs add and here.